components/List/CategoriesList.vue

```diff
@@ -31,6 +31,12 @@ export default {
         ...mapState('list', {
             elements: state => state.elements,
         }),
+        elementsByGroupInLanguage() {
```

Comment:
Wouldn't be better to destructure this.elements[this.languageCode] ? elementsByLanguageCode

Reply:
I do not agree, because through the destructure array we specify the index to which we refer. here we do not know specifically under which index is `this.languageCode` variable.